umbraco/Umbraco-CMS
# Filter out disallowed content types from the delivery API content index

### Prerequisites


### Description

This PR ensures that we never add content to the delivery API content index, if it is of a type which is not allowed to output through the delivery API.

### Testing this PR

1. Create some published content of different types.
2. Rebuild the delivery API content index from Examine management.
3. Verify that all content has been added to the index.
4. Add one of the content types to `DisallowedContentTypeAliases` in appsettings.
5. Rebuild the delivery API content index from Examine management again.
6. Verify that no content of the disallowed type has been added to the index.
